src/tribol/geom/CompGeom.hpp

```diff
@@ -53,28 +53,6 @@ class ContactPlanePair : public CompGeomPair {
    */
   TRIBOL_HOST_DEVICE ContactPlanePair( InterfacePair* pair, const Parameters& params, const int dim );
 
-  /*!
-   * \brief check to see if face-pairs are interacting
-   *
-   * \param [in] mesh1 mesh data viewer for mesh 1
-   * \param [in] mesh2 mesh data viewer for mesh 2
-   *
-   * \return face geometry exception
-   */
-  TRIBOL_HOST_DEVICE virtual FaceGeomException checkFacePair(
```

Comment:
It looks like these functions weren't changed, just moved location in the file? Please confirm.